.github/workflows/check.yml

```diff
@@ -23,4 +23,4 @@ jobs:
         run: "pip install pre-commit"
 
       - name: "Run checks with pre-commit"
-        run: "pre-commit run --all-files --show-diff-on-failure --color always"
+        run: "make precommit"
```

Comment:
Maybe `make lint`? I don't know if the Makefile is necessary: the idea is to run via git hook. But, I'm also not strongly opposed, so fine to merge it as is.

Reply:
Since I'm working on unit tests which aren't typically run by hook (the pre-commit team recommends against it, at least), I felt a Makefile would be useful going forward. Will change to `make lint`!

Reply:
1a8779b